<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lbJobListDirectory">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Job list directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pbJobListDir">
     <property name="text">
      <string>Choose...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbHwConfigFile">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Hardware configuration file</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pbHWConfigFile">
     <property name="text">
      <string>Choose...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbJobFileSubdirectory">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Job file subdirectory</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lbJobFileSubdirectoryName">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>data</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pbJobFileSubdir">
     <property name="text">
      <string>Choose...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="jle_resources.qrc">
         <normaloff>:/icons/Resources/pic/Button_NOK.png</normaloff>:/icons/Resources/pic/Button_NOK.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAccept">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="jle_resources.qrc">
         <normaloff>:/icons/Resources/pic/Button_OK.png</normaloff>:/icons/Resources/pic/Button_OK.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lbHwConfigFileName">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>data</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lbJoblistDirectoryName">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="jle_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbAccept</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbJobListDir</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>handlePbJobListDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbHWConfigFile</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>handlePbHWConfigFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbJobFileSubdir</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>handlePbJobFileSubDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handlePbJobListDir()</slot>
  <slot>handlePbHWConfigFile()</slot>
  <slot>handlePbJobFileSubDir()</slot>
 </slots>
</ui>
